spring-cloud-kubernetes-test-support/src/main/java/org/springframework/cloud/kubernetes/integration/tests/commons/fabric8_client/Util.java

```diff
@@ -104,11 +96,6 @@ public void createAndWait(String namespace, String name, @Nullable Deployment de
 						.get(0)
 						.setImage(imageFromDeployment + ":" + pomVersion());
 				}
-				else {
```

Comment:
I think this was needed for Jenkins because on our end we hit threshold limits from Dockerhub inside of K3S.  If we pull the images and load them in in the test iself we use our Dockerhub credentials and we don't run into the threshold limits from Dockerhub